src/components/dar_collection_table/DarCollectionTableCellData.js

```diff
@@ -107,9 +104,9 @@ export function institutionCellData({institution = '- -', darCollectionId, label
   };
 }
 
-export function datasetCountCellData({datasets = [], darCollectionId, label = 'datasets'}) {
+export function datasetCountCellData({datasetIds = [], darCollectionId, label = 'datasets'}) {
   return {
-    data: datasets.length > 0 ? datasets.length : '- -',
+    data: datasetIds.length > 0 ? datasetIds.length : '- -',
```

Comment:
optional nit: dar collection summaries have a `datasetCount` field that could be used instead